10-glossary.md

```diff
@@ -14,27 +16,56 @@ An issue is a problem or bug that needs to be addressed in the code. Issues can
 
 ## Clone
 
-Cloning is the process used to copy an existing Git repository into a new local directory.  The `git clone` command will create a new local directory for the repository, copy all the contents of the specified repository, create the remote tracked branches, and checkout an initial branch locally. By default, Git clone will create a reference to the remote repository called origin.
+Cloning is the process used to copy an existing Git repository into a new local directory. The `git clone` command will create a new local directory for the repository, copy all the contents of the specified repository, create the remote tracked branches, and checkout an initial branch locally. By default, Git clone will create a reference to the remote repository called origin.
 
 ## Fork
 
-A fork is a copy of a repository. When you fork a repository, you create a new copy of the codebase that you can modify and experiment with without affecting the original codebase. 
+A fork is a copy of a repository. When you fork a repository, you create a new copy of the codebase that you can modify and experiment with without affecting the original codebase.
 
 ## Branch
 
 A branch is a separate version of the code that's created for development purposes. Branches allow contributors to experiment with changes without affecting the main codebase. When changes are ready to be merged into the main codebase, they're typically submitted as a pull request.
 
 ## Pull Request
 
-A pull request is a request from a contributor to a maintainer for changes made to the code to be pulled into a codebase. 
+A pull request is a request from a contributor to a maintainer for changes made to the code to be pulled into a codebase.
 
 ## Merge
 
 Merging is the process of combining changes from one branch into another. When a pull request is accepted and merged, the changes made in the pull request become part of the main codebase.
 
 ## Docs
- This is an abbreviation for the word documentation, a piece of text that provides information on how to contribute to OSS projects and expectations for contributors. Markdown, a text-based coding language is  often used to create these forms of content. Want to see one in action? Check out the [Style Guide in our Pizza Verse repo](https://github.com/open-sauced/pizza-verse/blob/main/style-guide.md)!
+
+This is an abbreviation for the word documentation. Its primarily use is to explain how to implement and use the product. Documentation also provides people information on how to contribute to OSS projects and expectations for contributors. Markdown, a text-based coding language is often used to create these forms of content. Want to see one in action? Check out the [Style Guide in our Pizza Verse repo](https://github.com/open-sauced/pizza-verse/blob/main/style-guide.md)!
```

Comment:
In the sentence: "Its primarily use is to explain how to implement and use the product," there's a small typo. The word "primarily" should be "primary."

Correct sentence: "Its primary use is to explain how to implement and use the product."

Reply:
Suggested changes are done. I have also made one change on the next sentence. Let me know if that is valid according to you.